life-sync.cabal

```diff
@@ -13,7 +13,7 @@ stability:           experimental
 build-type:          Simple
 cabal-version:       2.0
 tested-with:         GHC == 8.2.2
-                   , GHC == 8.4.3
+                   , GHC == 8.4.4
```

Comment:
I guess, you can update Travis as well to reflect this change:
https://github.com/kowainik/life-sync/blob/7e889f3038a72a365d7fb860158f6a45c297c8ec/.travis.yml#L28-L39